### PR TITLE
tools/scylla-nodetool: add doc link for getsstables and sstableinfo commands

### PR DESCRIPTION
**Pull request by denesb**
Just like all the other commands already have it. These commands didn't have documentation at the point where they were implemented, hence the missing doc link.

The links don't work yet, but they will work once we release 6.0 and the current master documentation is promoted to stable.

### CODE REVIEW

File: tools/scylla-nodetool.cc [hunk 1]
```diff
@@ -3438,6 +3439,7 @@ Fore more information, see: https://opensource.docs.scylladb.com/stable/operatin
                 "sstableinfo",
                 "Information about sstables per keyspace/table",
 R"(
+Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/sstableinfo.html
```
Comment (avikivity) [inline tools/scylla-nodetool.cc:3442]: s/Fore/For/. 

We need to make the links refer to the version for when they were built.

Let's have a doc_link() function. fmt::format("Fore more information, see {}", doc_link("operating-scylla/nodetool-commands/sstableinfo.html"). The doc_link() function can use information from SCYLLA-VERSION-GEN (via release.cc) to get the rest of the URL and the version reference. For later of course.

/cc @annastuchlik

Comment (tzach) [reply tools/scylla-nodetool.cc:3442]: Another alternative is using the same source to generate both docs and the CLI --help output
Here is one way to do so
https://github.com/scylladb/scylla-manager/pull/2987

Comment (denesb) [reply tools/scylla-nodetool.cc]: Fixed the typo. I also fixed it for the other commands, copy-pasta spreads fast.

Comment (denesb) [reply tools/scylla-nodetool.cc]: As for the improvement, I will follow-up with the suggested `doc_link()` method.

**Comment by denesb** [comment]
@scylladb/scylla-maint ping. There is an open comment but it resulted in a follow-up, there is no action-item left for this PR.

**Review by nyh — APPROVED** [review]
I'll merge since all other commands already have this link, but I want to raise some doubt (that you can consider later): Why is this link the correct one? What if someone is running an old version of nodetool, or Enterprise version - and this link isn't the right one? Why do we need this link at all?

**Comment by denesb** [comment]
> I'll merge since all other commands already have this link, but I want to raise some doubt (that you can consider later): Why is this link the correct one? What if someone is running an old version of nodetool, or Enterprise version - and this link isn't the right one?

This is exactly what the follow-up agreed upon above is about: generate the link such that it refers to the current product and version. I have a WIP branch with this, will try to submit soon.

> Why do we need this link at all?

I wanted each command to only have a short description in the inline help, referring to the online docs for more details and examples.

**Pull request merged**